tests/integration/hdf5/test_ecephys.py

```diff
@@ -25,7 +25,8 @@ def setUpContainer(self):
         eg = ElectrodeGroup(name='elec1',
                             description='a test ElectrodeGroup',
                             location='a nonexistent place',
-                            device=self.dev1)
+                            device=self.dev1,
+                            position=[(1., 2., 3.), ])
```

Comment:
One question - why is this a list at all? Is there a case where we would have multiple tuples in this list? What would that represent?

Reply:
I also wasn't sure what the use case of the list would be - I added a test case with multiple tuples for now but not sure if that is necessary?